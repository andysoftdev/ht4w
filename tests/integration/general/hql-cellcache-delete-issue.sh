#!/usr/bin/env bash

HT_HOME=${INSTALL_DIR:-"$HOME/hypertable/current"}
SCRIPT_DIR=`dirname $0`

trap "$HT_HOME/bin/ht-stop-servers.sh" EXIT

$HT_HOME/bin/ht-start-test-servers.sh --clear --no-thriftbroker --Hypertable.RangeServer.Maintenance.Interval 100

cat $SCRIPT_DIR/hql-cellcache-delete-issue.hql | $HT_HOME/bin/ht shell --test-mode >& hql-cellcache-delete-issue.output

diff $SCRIPT_DIR/hql-cellcache-delete-issue.golden hql-cellcache-delete-issue.output
if [ $? -ne 0 ]; then
    echo "error: diff returned non-zero, exiting..."
    exit 1
fi

exit 0

