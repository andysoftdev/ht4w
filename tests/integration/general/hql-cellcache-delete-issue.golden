CREATE NAMESPACE cellcache_delete_issue;
USE cellcache_delete_issue;
DROP TABLE IF EXISTS AUTHSETTINGS;
CREATE TABLE AUTHSETTINGS (
USERSETTINGS MAX_VERSIONS 10 COUNTER false,
PASSWORD MAX_VERSIONS 10 COUNTER false,
ACCESS GROUP AG1 (USERSETTINGS, PASSWORD)
);
INSERT INTO AUTHSETTINGS VALUES ("2015-09-24 21:06:01.000000000","praveen.mishra@shriramcity.com","USERSETTINGS:is-deactivated-user","0"),("2015-09-24 21:06:01.000000000","praveen.mishra@shriramcity.com","PASSWORD","[j{k'642cr?");
COMPACT RANGES ALL;
pause 5;
DELETE * FROM AUTHSETTINGS WHERE ROW="praveen.mishra@shriramcity.com";
SELECT PASSWORD,USERSETTINGS FROM AUTHSETTINGS WHERE  ( CELL = "praveen.mishra@shriramcity.com","USERSETTINGS:is-deactivated-user" OR CELL = "praveen.mishra@shriramcity.com","PASSWORD" )  DISPLAY_TIMESTAMPS MAX_VERSIONS 1 NO_CACHE;
